test/test_images/helloserviceaccount/service.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sa-volume
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+      - # This is the Go import path for the binary to containerize
+        # and substitute here.
+        image: ko://knative.dev/serving/test/test_images/helloserviceaccount
+        volumeMounts:
+        - name: foo
+          mountPath: /hello/
+      volumes:
+      - name: foo
+        projected:
+          sources:
+          - secret:
+              name: bar
+          - configMap:
+              name: baz
```

Comment:
Shouldn't this actually mount a ServiceAccountToken?

Reply:
We could this is actually set in the test: TestProjectedServiceAccountToken